tedana/decomposition/pca.py

```diff
@@ -172,7 +173,7 @@ def tedpca(
     See Also
     --------
     :func:`tedana.utils.make_adaptive_mask` : The function used to create
-        the ``adaptive_mask` parameter.
+        the `adaptive_mask` parameter.
```

Comment:
Single backticks italicize in RST, while double backticks indicate code formatting, so I think you should add a backtick to fix the issue, rather than removing one.

Reply:
Just changed to double backticks.